app/services/ui_notifications/clean_expired.rb

```diff
@@ -0,0 +1,30 @@
+module UINotifications
+  class CleanExpired
+    def initialize(options = {})
+      blueprint = options[:blueprint]
```

Comment:
you can simplify these a bit by using kwargs, something like `def initialize(blueprint: nil, group: nil, expired_at: Time.now.utc)`